go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	github.com/pelletier/go-toml v1.6.0 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/rogpeppe/go-internal v1.5.1
+	github.com/rogpeppe/godef v1.1.2 // indirect
```

Comment:
I'm not sure about the `go.mod` and `go.sum` changes.  Are they relevant to this PR?

Reply:
No... when I built hugo, the build system made the changes to this file. Should I undo them?

Reply:
I would undo the changes to the go.* files and then rebuild to see if they are added back.  If not, remove them from the PR.